### PR TITLE
Added projects section UI

### PR DESCRIPTION
**Pull request by BhavyaSheth22**
The projects section opens up with the categories page where various domains are listed.
Screenshot from 2020-12-21 18-32-30

On clicking on a category, the required projects are listed with title, domain, image and description details.
Screenshot from 2020-12-21 18-35-40

### CODE REVIEW

**Review by ShubhankarKG — COMMENTED** [review]
Did not review the server part yet, will get to that after we finish on the models and db data.

**Comment by BhavyaSheth22** [comment]
> Did not review the server part yet, will get to that after we finish on the models and db data.

I have cleaned up the unwanted code. Please let me know once the data and models are available.

**Comment by ShubhankarKG** [comment]
We've merged #53 just now, which has the necessary controllers and routes for projects. Could you please rebase your branch to adapt to the changes we made? :pray:  

Thanks.

File: new_client/src/components/projects/Projects.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        axios
+            .get(process.env.REACT_APP_API + "/projects")
+            .then((res) => setDomains(res.data));
+    }, [])
+
+    return (
+        <Container maxWidth="lg">
+        <Grid container spacing={1}>
+        <Grid item xs={12}>
+        <Typography align="center" variant="h4" gutterBottom="true"> Project Categories </Typography>
```
Comment (ShubhankarKG) [inline new_client/src/components/projects/Projects.jsx:19]: `gutterBottom` is a boolean prop, so only true (without the "") or just `gutterbottom` should suffice.

File: new_client/src/components/projects/Projects.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        </Grid>
+        {domains.map((group) => (
```
Comment (ShubhankarKG) [inline new_client/src/components/projects/Projects.jsx:21]: We should add an error boundary here, like 
```jsx
{ domains.length > 0 && domains.map((group) => ())
}
```

File: new_client/src/components/projects/ProjectList.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                if(res.status == 200)
+                    setProjects(res.data);
+            });
+    }, [])
+
+    return (
+        <Container maxWidth="lg">
+        <Grid container spacing={1}>
+        <Grid item xs={12}>
+        <Typography align="center" variant="h4" gutterBottom="true"> {`${category} projects`} </Typography>
```
Comment (ShubhankarKG) [inline new_client/src/components/projects/ProjectList.jsx:24]: `gutterBottom` is a boolean prop, so only true (without the "") or just `gutterbottom` should suffice.

**Comment by BhavyaSheth22** [comment]
Sorry for the late response. I have completed the requested changes. There is some seed data in the db, and this is what the UI looks like:
Screenshot from 2021-01-20 10-10-52
Screenshot from 2021-01-20 10-11-57

**Review by ShubhankarKG — APPROVED** [review]
The changes look good to me, thanks! Let's wait for @Saif807380 to have a look.

**Comment by Saif807380** [comment]
LGTM. Great Work!

**Pull request merged**